### DISCUSSION TITLE
Latest version number

### DISCUSSION THREAD

**Discussion by ijuedt** [wixtoolset/issues#7899] [WiX Development]
When compiling the latest code (after fix for "Build tweaks to support VS 17.8.."), the binary built presents itself as a "5.0.0-preview" build in --help. Is this by intent? I'd like to manage my local binaries under the appropriate version number.

**Comment by chrpai** [comment]
Have you looked at wix/src/version.txt ?   It has 5.0.0-preview in it because that's what the develop branch represents today.

I can't speak with authority on what the maintainers intents were but generally I expect that it's a function of WiX being open sourced under a reciprocal license and a desire to make sure that improvements are contributed back.
